### ISSUE TITLE
Unsupported spread operator in iOS Safari and Edge

### ISSUE THREAD

**Issue by lokhmakov** [kettanaito/atomic-layout#206]
## What:

iOS: Unexpected token '...'. Expected a property name. 
Edge: SCRIPT1028: Expected identifier, string or number

## Why:

Because spread operator does not supported.

## Environment:

- atomic-layout version: `0.9.4`

## Steps to reproduce:

Any code that import `atomic-layout`.

## Screenshots:

<img width="322" alt="Screen Shot 2019-07-25 at 11 27 36" src="https://user-images.githubusercontent.com/94939/61858783-74a5d880-aecf-11e9-8e8a-5059de390577.png">
<img width="963" alt="Screen Shot 2019-07-25 at 11 28 41" src="https://user-images.githubusercontent.com/94939/61858789-77083280-aecf-11e9-86a5-4f658366a6fe.png">

## Solution:

After rebuild `npx babel ATOMIC_LAYOUT/esm/index.js > ATOMIC_LAYOUT/esm/index.js` with 

```
[
  '@babel/plugin-proposal-object-rest-spread',
  { loose: true, useBuiltIns: true },
],
```

All works fine.

**Comment by kettanaito** [comment]
Hi, @lokhmakov. Thank you for reporting this. 

Would you mind adjusting Rollup config for ESM module build? Should be a trivial operation. I can support you during the pull request had you had any questions.

**Comment by kettanaito** [comment]
Meanwhile, a note to myself: we should have a test suite that verifies this.

Currently each library build is included in the respective Storybook build, and then all E2E tests are run against that Storybook. Evidently that ensures working condition only in Chrome (since Cypress, which we use for E2E tests, runs on Chrome).

**Comment by lokhmakov** [comment]
As I found, right now rollup ESM build do not use babel plugins section at all. For proper build we need change only one line:

```js
babel({ ...babelConfig, extensions: ['.ts', '.tsx'] }),
```

I have created local branch `206-unsupported-ios-edge` with commit `Rollup - ESM build - Fix babel plugins` and ready to push.

**Comment by kettanaito** [comment]
Nice catch! Does this also mean that babel compilation doesn't use plugins for UMD module as well? 

In that case we should integrate `extensions` change to be used with any babel bundling:

```js
// rollup.config
import useBabel from 'rollup-plugin-babel'

const babel = (overrides = {}) => {
  return useBabel({
    ...babelConfig,
    ...overrides,
    extensions: ['.ts', '.tsx'],
  })
}

const anyBuild = {
  plugins: [babel()],
}
```

**Comment by lokhmakov** [comment]
Yep, all build do not use plugins section.

**Comment by kettanaito** [comment]
Published under `0.9.5`.

**Comment by hallaji** [comment]
Apparently, the `cjs` and `umd` bundles are exposing spread operator which results in an error on a bit old browsers such as Edge.
